lms/scripts/__init__.py

```diff
@@ -1,3 +1,2 @@
 from lms.scripts.devdata import devdata
-
-__all__ = ["devdata"]
+from lms.scripts.initdb import initdb
```

Comment:
The other approach I've seen to this is to have all the scripts have `main` so the target is `lms.scripts.initdb:main`